cmux-tui/docs/PR-INTENT-BOARD.md

```
@@ -1,6 +1,46 @@
+| [#10891](https://github.com/manaflow-ai/cmux/pull/10891), Lawrence Chen | `e16aa8c35bbb1fafa7b3cb1340f872754c66d6a7` on stale `642a65b1512d0d61aaef88290f90ef3408bbee74` | [33104968098](https://github.com/manaflow-ai/cmux/actions/runs/33104968098) queued; [33104965438](https://github.com/manaflow-ai/cmux/actions/runs/33104965438) in progress | Mergeability unknown; five CodeRabbit reviews apply to earlier heads |
+
+Closed without merge: [#9806](https://github.com/manaflow-ai/cmux/pull/9806)
+`406529665e5494ca559acab47079d8e7fb274386`, [#9813](https://github.com/manaflow-ai/cmux/pull/9813)
+`3b8d500aa23cfe9a7fbbe4a1dbdcf1be19902c61`, [#10136](https://github.com/manaflow-ai/cmux/pull/10136)
+`0786b6b37e5a397c1acc15b14be4a89f4363117b`, [#10413](https://github.com/manaflow-ai/cmux/pull/10413)
+`891544e0ab1f1ab277213b984e7f53078374fb63`, [#10237](https://github.com/manaflow-ai/cmux/pull/10237)
+`187dffe3e181fd6a85f99dc3fec2244c4fbe6fff`, [#10267](https://github.com/manaflow-ai/cmux/pull/10267)
+`7c8e4130737cf15f81086603364b587b13c05f40`, and [#10746](https://github.com/manaflow-ai/cmux/pull/10746)
+`9fa4c1497719f3c205ce6d402b3ce338d7fd5504`. They did not change main, so no
+rollback applies. Their replacements are #10134, #10259, #10521, #10263,
+#10268, and merged [#10985](https://github.com/manaflow-ai/cmux/pull/10985).
```

markdownlint-cli2 (0.23.2)

[warning] No space after hash on atx style heading

(MD018, no-missing-space-atx)